src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
I see. But that seems to be a very particular case. The user can still override `_getChildView` for those corner cases. For the vast majority of people this would not be a problem.

The PR has been updated with the docs.